airbyte-webapp/src/components/TitleBlock/TitleBlock.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+import { H5 } from "components";
+import ContentCard from "components/ContentCard";
+
+interface IProps {
+  title: React.ReactElement;
+  actions: React.ReactElement;
+}
+
+const TitleBlockComponent = styled(ContentCard)`
```

Comment:
We are no longer using Styled components for styling. Could you please convert this into SASS modules which we are moving to.